packages/frontend/src/components/NavBar/index.tsx

```diff
@@ -85,8 +89,27 @@ const NavBar = () => {
                         errorLogs={errorLogs}
                         setErrorLogsVisible={setErrorLogsVisible}
                     />
-                    <NavHeading>{user.data?.organizationName}</NavHeading>
-
+                    <ProjectDropdown
+                        disabled={isLoading || (data || []).length <= 0}
+                        options={data?.map((item) => ({
+                            value: item.projectUuid,
+                            label: item.name,
+                        }))}
+                        fill
+                        value={selectedProjectUuid}
+                        onChange={(e) => {
+                            showToastSuccess({
```

Comment:
Not sure we need a toast if the whole screen changes to another project.

Reply:
cc: @PriPatel

Reply:
sounds good!